mods/parsoid.js

```diff
@@ -809,6 +852,37 @@ module.exports = function(options) {
                 body: {
                     valueType: 'json'
                 }
+            },
+            // stashing resources for HTML, wikitext and data-parsoid
+            {
+                uri: '/{domain}/sys/key_rev_value/parsoid.stash.html',
+                body: {
+                    revisionRetentionPolicy: {
+                        type: 'latest',
+                        count: 1,
+                        grace_ttl: 86400
+                    },
+                    valueType: 'blob',
+                    version: 1
+                }
+            },
+            {
+                uri: '/{domain}/sys/key_rev_value/parsoid.stash.wikitext',
+                body: {
+                    valueType: 'blob'
+                }
+            },
+            {
+                uri: '/{domain}/sys/key_rev_value/parsoid.stash.data-parsoid',
+                body: {
+                    revisionRetentionPolicy: {
+                        type: 'latest',
+                        count: 1,
```

Comment:
@d00rman I've written a test, and then I didn't want through it away, one after another, so now it will work both in cassandra and SQLite (https://github.com/wikimedia/restbase-mod-table-cassandra/pull/156 & https://github.com/wikimedia/restbase-mod-table-spec/pull/20 )